plugins/woocommerce/tests/e2e-pw/tests/shopper/checkout-create-account.spec.js

```diff
@@ -69,6 +70,21 @@ test.describe( 'Shopper Checkout Create Account', () => {
 		await api.put( 'payment_gateways/cod', {
 			enabled: true,
 		} );
+
+		// make sure there's no pre-existing customer that has the same email we're going to use for account creation
+		const { data: customersList } = await api.get( 'customers', {
+			email: billingEmail,
+		} );
+
+		if ( customersList && customersList.length ) {
+			const customerId = customersList[ 0 ].id;
+
+			console.log(
+				`Customer with email ${ billingEmail } exists! Deleting it before starting test...`
+			);
+
+			await api.delete( `customers/${ customerId }`, { force: true } );
+		}
```

Comment:
Without this, retries of this test will certainly fail because each retry will use the same email when creating an account during checkout.